include/crm/common/ipc.h

```diff
@@ -150,6 +150,10 @@ enum crm_ipc_flags
     //! All replies to proxied connections are sent as events.  This flag
     //! preserves whether the events should be treated as an Event or a Response
     crm_ipc_proxied_relay_response  = (UINT32_C(1) << 18),
+    //! This is a multi-part IPC message
```

Comment:
All of these internal ones could have doxygen like 
```
/*!
  * \internal
  * \brief [Description]
  */
```

if we care enough. The "internal use only" comment isn't going to show up in the generated docs currently. But it doesn't really matter.

(There also may be a way to do it for all of them with a group directive.)

Reply:
We can address this in a separate PR later, since we need to deal the already existing ones too.